plugin.video.twit/changelog.txt

```diff
@@ -0,0 +1,157 @@
+Version 2.1.8
```

Comment:
please drop from submission. Changelog is no longer parsed by kodi in favor of the `<news>` tag